spec/controllers/v0/yellow_ribbon_programs_controller_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V0::YellowRibbonProgramsController, type: :controller do
+  context 'when determining version' do
+    it 'uses a production version as a default' do
+      create(:version, :production)
+      preview = create(:version, :preview)
+      create(:yellow_ribbon_program)
+      create(:yellow_ribbon_program, version: preview.number)
+      get(:index)
+      body = JSON.parse response.body
+      # we have one record associated with the production version, so we expect one result.
+      # this fails because we are getting back both the production and preview records
+      expect(body['data'].count).to eq(1)
```

Comment:
It's failing.  😄 
You need to account for the version in your `YellowRibbonProgram.search`.
You might want to put back the `version` scope that you just removed.  I commented on it because it wasn't being used anywhere, but it looks like this is where it _should_ be used.
Something like `YellowRibbonProgram.version(version).search(params)`

Reply:
Updated! 🙂